bin/run_ffi_tests.dart

```diff
@@ -134,14 +134,15 @@ final tests = [
 ];
 
 void main(List<String> args) {
-  final engineDir = '../dart-sdk/sdk';
-  final compilerBuildDir = 'xcodebuild/DebugARM64';
-  final runtimeBuildDir = 'xcodebuild/DebugSIMARM64';
-  final runtimeBuildDirPath = p.absolute('$engineDir/$runtimeBuildDir');
-  final compilerBuildDirPath = p.absolute('$engineDir/$compilerBuildDir');
+  // Relative to this script's working directory.
+  final sdkDir = '../dart-sdk/sdk';
+  final compilerConf = 'DebugARM64';
+  final runtimeConf = 'DebugSIMARM64';
```

Comment:
These could be `const`

Reply:
Good call. For consistency we could use the same analysis_options as in `shorebird_cli` and other packages

Reply:
Done, thanks for the suggestion.